<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>531</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Comic Sans MS</family>
   </font>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Youtube2Mp3 by dvtq</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Images/icon.jpg</normaloff>Images/icon.jpg</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>35</width>
    <height>35</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>256</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="thumbPreviewLabel">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>320</width>
             <height>256</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>Images/default_thumb.jpg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="openExternalLinks">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="thumbnail_label">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Video thumbnail&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="insertLinkLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Insert Youtube link here:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignTop">
         <widget class="QLineEdit" name="linkInputLine">
          <property name="minimumSize">
           <size>
            <width>450</width>
            <height>24</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Insert link here!</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="videoModeBt">
            <property name="text">
             <string>Video mode</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="playlistModeBt">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="toolTip">
             <string>Playlist mode!</string>
            </property>
            <property name="text">
             <string>Playlist mode</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="clearButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="getInfoButton">
            <property name="text">
             <string>Get Info</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="downloadButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>MS Shell Dlg 2</family>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Download!</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Images/download-cloud-flat.png</normaloff>Images/download-cloud-flat.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Save To:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLineEdit" name="fordelNameLine">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="toolButton">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="infoLabel">
          <property name="font">
           <font>
            <family>Comic Sans MS</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Video Info:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="titleLabel">
            <property name="text">
             <string>Title:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="titleLine">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignJustify|Qt::AlignVCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="authorLabel">
            <property name="text">
             <string>Channel:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="authorLine">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="viewsLabel">
            <property name="text">
             <string>Views:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="viewLine">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lengthLabel">
            <property name="text">
             <string>Length:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lengthLine">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="publishLabel">
            <property name="text">
             <string>Publish day:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="publishLine">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;Song Metadata:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="scaledContents">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="titleSongLb">
            <property name="text">
             <string>Title:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="titleSongLine">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="artistSongLb">
            <property name="text">
             <string>Artist:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="artistSongLine"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="bitrateSongLb">
            <property name="text">
             <string>Bitrate:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>24</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="countProcess">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;0/1:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="bitrateSongLine"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resourceFile.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>playlistModeBt</sender>
   <signal>clicked()</signal>
   <receiver>titleLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playlistModeBt</sender>
   <signal>clicked()</signal>
   <receiver>authorLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>396</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playlistModeBt</sender>
   <signal>clicked()</signal>
   <receiver>viewLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>420</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playlistModeBt</sender>
   <signal>clicked()</signal>
   <receiver>lengthLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>339</x>
     <y>456</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playlistModeBt</sender>
   <signal>clicked()</signal>
   <receiver>publishLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>384</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>485</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>videoModeBt</sender>
   <signal>clicked()</signal>
   <receiver>titleLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>351</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>videoModeBt</sender>
   <signal>clicked()</signal>
   <receiver>authorLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>423</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>350</x>
     <y>387</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>videoModeBt</sender>
   <signal>clicked()</signal>
   <receiver>viewLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>439</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>videoModeBt</sender>
   <signal>clicked()</signal>
   <receiver>lengthLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>386</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>450</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>videoModeBt</sender>
   <signal>clicked()</signal>
   <receiver>publishLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>155</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playlistModeBt</sender>
   <signal>clicked()</signal>
   <receiver>titleSongLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>378</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playlistModeBt</sender>
   <signal>clicked()</signal>
   <receiver>artistSongLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>432</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>414</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playlistModeBt</sender>
   <signal>clicked()</signal>
   <receiver>bitrateSongLine</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>417</x>
     <y>168</y>
    </hint>
    <hint type="destinationlabel">
     <x>520</x>
     <y>443</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>playlistModeBt</sender>
   <signal>clicked()</signal>
   <receiver>progressBar</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>479</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
